aggregation.py

```diff
@@ -22,13 +22,18 @@ class SafeAgregation():
     """
     default_measure_types = MEASURE_TYPES.union({max_percentage})
 
-    def __init__(self, columns_apply_secret, measure_types=default_measure_types):
+    def __init__(self,  common_column: str, columns_apply_secret: list, measure_types=default_measure_types, *args, **kwargs):
         with open("config.json") as f:
             config = json.load(f)
         self.relevant_column = columns_apply_secret
         self.measure_types = measure_types
         self.rules_list = config["RULES"]
         self.dict_aggreg = self._create_dict_aggregation(columns_apply_secret)
+        self.common_column = common_column
+        self.frequency_threshold = next(item for item in self.rules_list if item["RULE_NAME"] == "FREQUENCY")[
+            'THRESHOLD']
+        self.dominance_threshold = next(item for item in self.rules_list if item["RULE_NAME"] == "DOMINANCE")[
+            'THRESHOLD']
```

Comment:
on ne peut pas faire un dictionnaire de rules_list plutôt ?

Reply:
self.rules_list = ... ?
garder fréquence et dominance séparée c'est plus simple pour la suite, car ils se font dans deux fonctions séparées, mais si jamais on vient à en rajouter plusieurs ça peut être pertinent de faire une liste, mais pour l'instant je ne sais pas si c'est très utile

Reply:
Y a t il un intérêt réel à définir règles de fréquence et de dominance dans le même objet ?
Si ce n'est pas le cas on pourrait faire beaucoup plus simple :
```suggestion
    self.frequency_treshold = item["FREQUENCY"]
    self.dominance_treshold = item["DOMINANCE"]
```
Où item serait du style :
```
        item = {
            "FREQUENCY": [rule1, rule2, ..., ruleN],
            "DOMINANCE": [rule1, rule2, ..., ruleN]
        }
```